Library/build.gradle

```diff
@@ -75,6 +75,7 @@ android {
 
         testImplementation "org.mockito:mockito-core:${versions.mockito2}"
         testImplementation "com.nhaarman.mockitokotlin2:mockito-kotlin:${versions.mockitokotlin}"
+        testImplementation "org.jetbrains.kotlinx:kotlinx-coroutines-test:${versions.kotlinx}"
```

Comment:
Added this so I can use `newSingleThreadContext` to force single-threaded execution, so the tests don't run in random order